.changeset/techdocs-improve-tests.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/techdocs-common': patch
+---
+
+Improved tests fetchTechDocsMetadata to AzureBlobStorage.
```

Comment:
Tests don't actually need changesets, but it makes sense to mention the improved error handling when the metadata can't be found